tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,17 +50,20 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
-        """invokes screen capture event, crops the status bar, and saves to the file"""
+    # hook to override default locale setting
+    # see https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L1019
+    def modify_settings(self, settings):
+        settings['language.current'] = self.locale
```

Comment:
Second line note, so you get notifications correctly. Actually, I misread this. This is your test override, I thought this was in core harness. My bad on that. This might be OK as-is.

FWIW, I think moving the locale selection from cmdline into the main gaiatest would be entirely reasonable (that's why I thought you were doing so here) in which case my comment would stand.